static/js/zamboni/collections.js

```diff
@@ -640,4 +640,9 @@ $(document).ready(function () {
     if ($('#details-edit').length && $('div.notification-box').length) {
         $(document).scrollTop($("div.primary").position().top);
     }
+
+    var notificationBox = document.querySelector('.notification-box');
```

Comment:
if we add `margin-bottom: 5px` to `.notification-box` , it doesn't increase the blue header part. It's only increasing the white area below the blue header.